site/en/rules/deploying.md

```diff
@@ -230,6 +258,32 @@ can either be in the standard location for the language the rules are for or a
 It is useful to users to have an `examples/` directory that shows users a couple
 of basic ways that the rules can be used.
 
+### Test module for the Bazel Central Registry *
+
+All rules available as a [Bazel module](https://bazel.build/docs/bzlmod#modules)
+from the [Bazel Central Registry (BCR)](https://github.com/bazelbuild/bazel-central-registry)
+should provide a Bazel test module under `tests/bcr/` that references the main
```

Comment:
```suggestion
must provide a Bazel test module under `tests/bcr/` that references the main
```

I think BCR should enforce this (because how else would they verify that a module is "good"?)

@Wyverald @meteorcloudy WDYT?

Reply:
Sure, for non-ruleset modules, the story might be different. I fully agree on that. But this guide is about rules so I think enforcing is preferable.

I don't feel too strongly about making this a `must`.

I'm wondering if it makes sense to use the existing `//examples` as test module?

Reply:
I don't think BCR should really enforce having a test module, though it's definitely recommended. So `should` is the right level of nudging, I think.